src/read/pe/file.rs

```diff
@@ -287,61 +401,21 @@ where
         Ok(imports)
     }
 
+    /// Note that this only returns "regular" exports.
```

Comment:
Maybe elaborate on "regular" exports for documentation? "regular" as opposed to what?

Reply:
Done, I've added a doc in the trait definition